package.json

```diff
@@ -27,17 +27,19 @@
     },
     "dependencies": {
         "@solana/wallet-standard-features": "^1.0.0",
-        "@solana/web3.js": "^1.58.0",
         "@wallet-standard/base": "^1.0.0",
-        "@wallet-standard/features": "^1.0.0",
-        "bs58": "^4.0.1",
-        "standard-version": "^9.5.0"
+        "@wallet-standard/features": "^1.0.0"
+    },
+    "peerDependencies": {
+        "bs58": "^5.0.0"
     },
     "devDependencies": {
+        "@solana/web3.js": "^1.91.0",
         "@types/bs58": "^4.0.1",
         "@types/node-fetch": "^2.6.2",
         "prettier": "^2.7.1",
         "shx": "^0.3.4",
+        "standard-version": "^9.5.0",
```

Comment:
what's this being used for?

Reply:
for yarn release

```
"release": "standard-version"
```

it was in dependencies, I moved to devDependencies